usaspending_api/agency/v2/views/agency_base.py

```diff
@@ -24,6 +24,15 @@ def toptier_code(self):
         # either a three or four digit numeric string based on the regex pattern in our route url.
         return self.kwargs["toptier_code"]
 
+    @cached_property
+    def agency_id(self):
+        from usaspending_api.references.models import Agency
```

Comment:
While this pattern can reduce imports when this code branch isn't reached, this file is already importing the module on line 17. So for clarity this `Agency` import statement should be combined above